.github/scripts/patch.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -x
+
+exit 1 # The following values must be change before every new patch release
```

Comment:
This was left on purpose and should be like this on purpose. It is because users need to specify which tag of an upstream dependency they need to patch. this will make sure that they change the file.

Reply:
I added better comments.